core/src/main/scala/cromwell/core/logging/LoggingTest.scala

```diff
@@ -0,0 +1,7 @@
+package cromwell.core.logging
+
+import java.util.regex.Pattern
+
+object LoggingTest {
+  def escapePattern(pattern: String) = Pattern.quote(pattern)
```

Comment:
Yes but I couldn't import it properly from anywhere when it was in the test folder..